libraries/AP_HAL_ChibiOS/hwdef/QioTekAdeptF407/hwdef.dat

```diff
@@ -0,0 +1,201 @@
+# hw definition file for processing by chibios_hwdef.py
+# for QioTekZealotF407 hardware from Qio-tek.com
+
+# MCU class and specific type
+MCU STM32F4xx CKS32F407xx
```

Comment:
Isn't this at odds with the other changes? I was expecting to see:
```
MCU CKS32F4xx CKS32F407xx 
```
With what you have here can you drop the STM32F4xx for instance

Reply:
you're right, thanks. fixed